zkp/src/verifying_key.rs

```diff
@@ -0,0 +1,17 @@
+use std::{env, io::Cursor};
+
+use ark_ec::mnt6::MNT6;
+use ark_groth16::VerifyingKey;
+use ark_mnt6_753::Parameters;
+use ark_serialize::CanonicalDeserialize;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    pub static ref ZKP_VERIFYING_KEY: VerifyingKey<MNT6<Parameters>> = {
+        let serialized = include_bytes!(concat!(env!("OUT_DIR"), "/verifying_keys.data"));
```

Comment:
Perhaps we should rename the file to `verifying_key.data` (instead of plural variant)